.github/workflows/post-usage.yml

```diff
@@ -9,23 +9,44 @@ on:
 jobs:
   Notify-Contributor-Requirements:
     runs-on: ubuntu-latest
-    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/run pipeline' }}
+    if: ${{ github.event.issue.pull_request && (github.event.comment.body == '/run pipeline' || github.event.comment.body == '/run pipeline ') }}
     steps:
       - name: Check Team Membership
         id: check_team
         run: |
-          set -e
-          curl -X GET -H "Authorization: Bearer ${{secrets.ORG_READER_GH_TOKEN}}" \
-          -H "Accept: application/vnd.github.v3+json" \
-          "https://api.github.com/orgs/${{github.event.organization.login}}/teams/${{inputs.required_team}}/memberships/${{github.actor}}"
+          set -euo pipefail
 
           # The '${{secrets.GITHUB_TOKEN}}' secret comes out of the box with GHA - See https://docs.github.com/en/actions/security-guides/automatic-token-authentication
           # Since this secret  does not have enough permissions to run below api we use the '${{secrets.ORG_READER_GH_TOKEN}}' which is an org level secret that we own
-          user_in_team=$(curl -X GET -H "Authorization: Bearer ${{secrets.ORG_READER_GH_TOKEN}}" \
-          -H "Accept: application/vnd.github.v3+json" \
-          "https://api.github.com/orgs/${{github.event.organization.login}}/teams/${{inputs.required_team}}/memberships/${{github.actor}}" | jq -r '.state')
+
+          API_URL="https://api.github.com/orgs/${{github.event.organization.login}}/teams/${{inputs.required_team}}/memberships/${{github.actor}}"
+          echo "Fetching the response for $API_URL"
+
+          api_response=$(curl -s -o response_body.json -w "%{http_code}" -X GET -H "Authorization: Bearer ${{secrets.ORG_READER_GH_TOKEN}}" \
+          -H "Accept: application/vnd.github.v3+json" "$API_URL")
+
+          # Log messages if API failure is observed
+          if [[ "$api_response" -ne 200 && "$api_response" -ne 201 ]]; then
```

Comment:
If as user is not found there is a 404 response. We want the message to be printed if the user is not found. I think it should print the message on any failure.

Reply:
Thanks for your input. I have made the changes to print error messages.